composer.json

```diff
@@ -29,44 +23,44 @@
         "doctrine/annotations": "^1.13",
         "eluceo/ical": "^0.15.0",
         "ezyang/htmlpurifier": "4.13.*",
-        "fideloper/proxy": "^4.4",
         "firebase/php-jwt": "^6.8",
         "fruitcake/laravel-cors": "^2.0",
         "glenscott/url-normalizer": "^1.4",
         "google/apiclient": "^2.2",
         "guzzlehttp/guzzle": "^7.0.1",
-        "laravel-doctrine/extensions": "1.4.*",
-        "laravel-doctrine/migrations": "2.3.*",
-        "laravel-doctrine/orm": "1.7.*",
-        "laravel/framework": "^v8.77.1",
+        "guzzlehttp/uri-template": "^1.0",
+        "laravel-doctrine/extensions": "1.5.2",
+        "laravel-doctrine/migrations": "3.0.1",
+        "laravel-doctrine/orm": "1.8.3",
+        "laravel/framework": "^9.0",
         "laravel/helpers": "^1.4",
         "laravel/tinker": "^2.5",
         "league/csv": "^9.6",
-        "league/flysystem-aws-s3-v3": "^1.0.29",
+        "league/flysystem-aws-s3-v3": "3.8.0",
         "league/oauth2-client": "^2.4",
         "muxinc/mux-php": "^3.11",
-        "php-opencloud/openstack": "dev-feature/guzzle_7_x",
+        "php-opencloud/openstack": "3.10.0",
         "pion/laravel-chunk-upload": "^1.4",
         "predis/predis": "v2.1.2",
         "ramsey/uuid": "^4.2",
         "s-ichikawa/laravel-sendgrid-driver": "~3.0",
         "simplesoftwareio/simple-qrcode": "^2.0",
         "sokil/php-isocodes": "^3.0",
-        "spatie/flysystem-dropbox": "^1.2",
+        "spatie/flysystem-dropbox": "^3.0.1",
         "stripe/stripe-php": "^7.95.0",
         "symfony/yaml": "^5.0",
         "tecnickcom/tcpdf": "^6.2",
-        "vladimir-yuldashev/laravel-queue-rabbitmq": "v11.3.*",
-        "zendframework/zend-math": "^3.2"
+        "vladimir-yuldashev/laravel-queue-rabbitmq": "13.3.5",
+        "laminas/laminas-math": "^3.7"
```

Comment:
please explain why "zendframework/zend-math" its needed for crypto and its a dep of other libs
this will break functionality

Reply:
Done. All references to Zend were changed to Laminas.